src/core/ready.js

```diff
@@ -11,7 +12,9 @@ var readyList = jQuery.Deferred();
 
 jQuery.fn.ready = function( fn ) {
 
-	readyList.then( fn );
+	readyList
+		.then( fn )
+		.catch( jQuery.readyException );
```

Comment:
Good point, it might be good to have a comment to that effect so nobody tries to "optimize" it later.